build.gradle

```diff
@@ -127,6 +127,7 @@ dependencies {
     compile group: 'com.github.javafaker', name: 'javafaker', version: '0.10'
 
     compile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
+    compile group: 'org.apache.commons', name: 'commons-text', version: '1.2'
```

Comment:
how big is this dependency?

Reply:
Just checked on my machine, `commons-text-1.2.jar` is 136,544 bytes (139 KB on disk)